sdk/src/test/java/com/horizen/utils/Ed25519Test.java

```diff
@@ -37,11 +41,14 @@ public void rfc_8032_test_1() {
         byte[] privateKey = Hex.decode(strPrivateKey);
         byte[] publicKey = Hex.decode(strPublicKey);
         byte[] signature = Hex.decode(strSignature);
-        byte[] signatureBC = new byte[64];
-        Ed25519.sign(privateKey, 0, publicKey, 0, (byte[]) null, msg, 0, msg.length, signatureBC, 0);
-        boolean isVerified = Ed25519.verify(signatureBC, 0, publicKey, 0, msg, 0, msg.length);
-        assertTrue(Arrays.equals(signature, signatureBC));
-        assertTrue(isVerified);
+        byte[] signatureBC = Ed25519.sign(privateKey, msg, publicKey);
```

Comment:
`rfc_8032_test_1` is not so clear name. Please, add some annotations about `rfc 8032`.
Is bouncy castle signing algorithm deterministic? so the same private key for the same message will generate the same signature without any random value chose inside?

Reply:
Done